raiden/network/transport/matrix.py

```diff
@@ -205,8 +206,9 @@ def start_health_check(self, node_address):
             # Ensure network state is updated in case we already know about the user presences
             # representing the target node
             self._update_address_presence(node_address)
-            assert self._get_room_for_address(node_address, allow_missing_peers=True),\
-                f'Could not get room for {node_address_hex!r}'
+            room = self._get_room_for_address(node_address, allow_missing_peers=True)
+            if not room:
+                self.log.warning('No room found or created for peer', peer=node_address_hex)
```

Comment:
this is strange, what will the app do if there's no room? In tests this got triggered on teardown, maybe we can ignore the error during the shutdown and assert otherwise.

Reply:
It shouldn't be an issue, if a `send_async` is triggered for a pair for which there's no room yet, it'll be created. It may only happen on teardown, but isn't fatal otherwise too.